Header header

# references
geometry_msgs/Vector3 accel_out
geometry_msgs/Vector3 ang_vel_out
